test/testdata/generate.sh

```diff
@@ -123,12 +119,9 @@ build_kb
 
 # Project version 2 uses plugin go/v2 (default).
 scaffold_test_project project-v2 --project-version=2
-scaffold_test_project project-v2-multigroup --project-version=2
-scaffold_test_project project-v2-addon --project-version=3 --plugins="go/v2,declarative"
 # Project version 3 (default) uses plugin go/v3 (default).
 scaffold_test_project project-v3
 scaffold_test_project project-v3-multigroup
 scaffold_test_project project-v3-addon --plugins="go/v3,declarative"
 scaffold_test_project project-v3-config --component-config
-scaffold_test_project project-v3-v1beta1
```

Comment:
I agree with removing the v2 projects since they are being deprecated, but are we also deprecating the v1beta1 support for the go/v3 plugin? If not then I think we should keep this one in.

Reply:
I agree with @everettraven.

Reply:
Yes, we do not want anybody using v1neta1. It does not work since K8s >= 1.22 and the flag is deprecated across all places

Reply:
Gotcha, then this is good.